### DISCUSSION TITLE
SplitButton left section should can open the items selector too?

### DISCUSSION THREAD

**Discussion by FabianMontoya** [primefaces/community#1206] [PrimeVue]
My propose is add the feature to open the item list selector also when we click in the left button side of the `SplitButton`, maybe with a prop and do the same that when we click the right section:

![Screenshot 2024-02-15 at 22 13 37](https://github.com/primefaces/community/assets/37662924/85fe306f-648e-49da-8ef3-0d320d3a0e55)

**Comment by HeuHax** [comment]
It would be nice if there was an option to merge the main button and the arrow into one.
